cmd/d2/main.go

```diff
@@ -204,9 +205,18 @@ func compile(ctx context.Context, ms *xmain.State, plugin d2plugin.Plugin, theme
 	if err != nil {
 		return nil, err
 	}
+	svg, err = imgbundler.InlineLocal(ms, svg)
```

Comment:
Put this in a new else of the png check below. No need to run this for png outputs.

Reply:
u do need to. InlineLocal and InlineRemote are mutually exclusive

Reply:
That's what I thought so why is InlineRemote running after InlineLocal? Shouldn't you only need to run one of them given like you said they're mutually exclusive.

Reply:
u always need to read local images off disk. localhost won't let u access them otherwise.

Reply:
Ohhh I see, my bad I misunderstood. makes sense.